File: JDK11/opentraceability/src/main/java/opentraceability/models/common/sbdhcontact.java
package opentraceability.models.common;

import opentraceability.Constants;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;

public class SBDHContact {

    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "Contact")
    public String ContactName = "";

    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "EmailAddress")
    public String EmailAddress = "";
}

File: Kotlin/OpenTraceability/src/main/kotlin/opentraceability/models/common/sbdhcontact.kt
package opentraceability.models.common

import opentraceability.Constants
import opentraceability.utility.attributes.OpenTraceabilityAttribute

class SBDHContact {

    @OpenTraceabilityAttribute(opentraceability.Constants.SBDH_NAMESPACE, "Contact")
    var ContactName: String = ""

    @OpenTraceabilityAttribute(opentraceability.Constants.SBDH_NAMESPACE, "EmailAddress")
    var EmailAddress: String = ""
}
